src/arrayMethodJoin.js

```diff
@@ -5,7 +5,24 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
just add default value - no need in additional variable
```suggestion
  [].__proto__.join2 = function(separator = ",") {
```